nginx-s3-gateway/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for nginx-s3-gateway.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/nginxinc/nginx-s3-gateway/nginx-oss-s3-gateway
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: unprivileged-oss-20221216
```

Comment:
I think if you are creating a new chart... you should use something a bit more recent ?